<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0K8EQOA" ignored="2" name="Default test(failed)" tests="1" failures="1" timestamp="2022-07-05T15:14:44 IST" time="0.065" errors="0">
  <testcase name="retry1" time="0.011" classname="com.testng.features.testing.TestNG_RetryAnalyser">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at com.testng.features.testing.TestNG_RetryAnalyser.retry1(TestNG_RetryAnalyser.java:10)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- retry1 -->
  <testcase name="retry2" time="0.0" classname="com.testng.features.testing.TestNG_RetryAnalyser">
    <ignored/>
  </testcase> <!-- retry2 -->
  <testcase name="retry3" time="0.0" classname="com.testng.features.testing.TestNG_RetryAnalyser">
    <ignored/>
  </testcase> <!-- retry3 -->
</testsuite> <!-- Default test(failed) -->
